### DISCUSSION TITLE
How do you keep EAV attributes synced across different environments (local/dev/staging/production)?

### DISCUSSION THREAD

**Discussion by elidrissidev** [OpenMage/magento-lts#2077] [Questions & Answers]
How do you go about keeping EAV attributes in sync across different environments (local/dev/staging/production), I've been doing it manually all this time and it's a pain. Do you guys have any way to automate this, maybe sql scripts in a custom module?

**Comment by kiatng** [comment]
For me, I create the attributes programmatically in an upgrade script so that it get deployed to staging and then production.
